app/assets/javascripts/application.js

```diff
@@ -11,7 +11,7 @@
 $(function () {
   $('a.preview').attr("target","_blank");
   $('form.preview').attr("target","_blank");
-  $(".select2").select2();
+  $(".select2").select2({ allowClear: true });
```

Comment:
Correct - that assignee one uses `allowClear` as well ([assignee_select.js](https://github.com/alphagov/publisher/blob/50d8ba6ebfb66c8d2bfe2415910cdfbf351ed568/app/assets/javascripts/modules/assignee_select.js#L5
)).

The only other select2-enabled components I could find (by searching for `select2` in `app/views`) are in that same template. Those are multi-selects and therefore already have a clear button.

(Even if there were any other affected selects, that would be okay since it only applies the button to dropdowns where the original developer explicitly set `include_blank: true`.)